### ISSUE TITLE
Question: Store files that do not appear in Game Materials for users

### ISSUE THREAD

**Issue by ClaudioWayne** [moloch--/RootTheBox#631]
Is it possible to store files that do net appear in Game Materials for users. I try to create hints that include screenshots. It would be counterproductive if the user could access the screenshot before purchasing the hint.

Thanks

Claudio

**Comment by eljeffeg** [comment]
I believe hints support markdown.  So, I think, you could essentially host the graphic wherever you like and just link to it, but just going off or memory and haven't tested it recently.

**Comment by ClaudioWayne** [comment]
Yes just dropped the files in the static folder and linked it via markdown. 
Works like charm.
Thx
